File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_varimp.R
# Train a Deep Learning model and validate on a test set
# and save the variable importances
model_vi <- h2o.deeplearning(
        x = x, 
        y = y, 
        training_frame = train,
        distribution = "multinomial",
        activation = "RectifierWithDropout", 
        hidden = c(200,200,200), 
        input_dropout_ratio = 0.2, 
        l1 = 1e-5, 
        validation_frame = test, 
        epochs = 10,
        variable_importances = TRUE)  #added

# Retrieve the variable importance
h2o.varimp(model_vi)

File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_varimp.py
# Train Deep Learning model and validate on test set
# and save the variable importances
model_vi = H2ODeepLearningEstimator(        
        distribution="multinomial",
        activation="RectifierWithDropout", 
        hidden=[32,32,32],
        input_dropout_ratio=0.2, 
        sparse=True,
        l1=1e-5, 
        epochs=10,
        variable_importances=True)

model_vi.train(
        x=x, 
        y=y, 
        training_frame=train, 
        validation_frame=test)

# Retrieve the variable importance
model_vi.varimp()
